VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -21,7 +21,7 @@
         <RelativeLayout
             android:id="@+id/announcement_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="fill_parent">
```

Comment:
THere's only 3 options (other than manually setting the dimensions): wrap_content, match_parent, or fill_parent.

Wrap_content causes the bug, and fill_parent is deprecated. If not match_parent, then what? Setting a numeric value?